cognee/api/v1/cognify/code_graph_pipeline.py

```diff
@@ -129,7 +136,7 @@ async def run_code_graph_pipeline(repo_path):
         Task(get_repo_file_dependencies),
         Task(enrich_dependency_graph, task_config={"batch_size": 50}),
         Task(expand_dependency_graph, task_config={"batch_size": 50}),
-        Task(summarize_code, summarization_model=SummarizedContent, task_config={"batch_size": 50}),
+        Task(summarize_code, summarization_model=SummarizedCode, task_config={"batch_size": 50}),
```

Comment:
Should the the summarize_code function have a summarization_model argument (as it seems to be always called with the SummarizedCode pydantic model)?

Reply:
Fair point, I can't think of a reason to keep it, so I'll remove it.

Reply:
done